test/e2e/conversion/conversion_test.go

```diff
@@ -87,12 +87,17 @@ func TestAPIConversion(t *testing.T) {
 		return false, fmt.Sprintf("%v", apiBinding.Status)
 	}, wait.ForeverTestTimeout, 100*time.Millisecond, "APIBinding never completed its initial binding")
 
-	t.Logf("Resetting the RESTMapper so it can pick up widgets")
-	mapper.Reset()
+	framework.Eventually(t, func() (success bool, reason string) {
+		t.Logf("Resetting the RESTMapper so it can pick up widgets")
+		mapper.Reset()
```

Comment:
methods like `mapper.RESTMapping` used by `helpers.CreateResourceFromFS` will invalidate the mapper (`mapper.Reset`) but only when the underlying cache is not fresh. We could rely on it, wdyt?

Reply:
That could work, but it won't work with a memCacheClient - you have to manually call reset.